src/Administration/Controller/AdministrationController.php

```diff
@@ -129,6 +136,28 @@ public function index(Request $request, Context $context): Response
         ]);
     }
 
+    private function readSharedResourceFile(string $relativePath): ?string
```

Comment:
why do we need to do that via PHP? :thinking:

Reply:
if have totally no clue how this all works :see_no_evil: but like this, it looks wrong :grin: 

don't we have also other assets that are shared and injected in both index files? 

btw: why do we have two of them? :sweat_smile: